doc/developer/topotests.rst

```diff
@@ -680,6 +680,28 @@ during the config_timing test.
 To specify different arguments for ``perf record``, one can use the
 ``--perf-options`` this will replace the ``-g`` used by default.
 
+Running Daemons under RR Debug (``rr record``)
+""""""""""""""""""""""""""""""""""""""""""""""
+
+Topotest can automatically launch any daemon under ``rr(1)`` to collect
+execute state. The daemon is run in non-daemon mode with ``rr record``.
+
+The execution state will be saved in the router specific directory
+(in a `rr` subdir that rr creates) under the test's run directoy.
+
+Here's an example of collecting ``rr`` execution state from ``mgmtd`` on router
+``r1`` during the config_timing test.
```

Comment:
``config_timing``

Reply:
> `config_timing`

not sure what's being asked for here :)

Reply:
I put double backtics but github rendered them; I'm saying config_timing should be code-formatted in the docs.